lib/puma/runner.rb

```diff
@@ -178,5 +178,18 @@ def ensure_output_directory_exists(path, io_name)
         raise "Cannot redirect #{io_name} to #{path}"
       end
     end
+
+    def stats
+      {
+        versions: {
+          puma: Puma::Const::PUMA_VERSION,
+          ruby: {
```

Comment:
It may be interesting to expose the whole `RbConfig::CONFIG` here if someone wants a complete environment information.

Reply:
I took a look, I don't think there's anything else in there that we really want to expose. Exposing all of that info would probably 10x the size of the stats hash, we do need to be conscious of that.